### PR TITLE
Velodrome documentation

### PR DESCRIPTION
**Pull request by apelisse**
Probably easier to review from there:

https://github.com/apelisse/test-infra/tree/velodrome-documentation/velodrome/

Also it would probably be easier if you directly edit readme and suggest change through a pull-request there rather than going back and forth a thousand times,

Thanks,

### CODE REVIEW

File: velodrome/fetcher/README.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+
+There is no set-up required as `fetcher` will create the github database if it
+doesn't exist, along with the various required tables.
+
+Design decision
+===============
+
+Because of the size of the kubernetes/kubernetes repository:
+
+- Accessing comments API doesn't work and returns `500 Internal Error` all the
```
Comment (spxtr) [inline velodrome/fetcher/README.md]: Which API calls are doing this?

File: velodrome/grafana-stack/README.md [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
 - Grafana as the front-end/display
-- Nginx is use as a proxy (mostly to fix CORS issue)
+- Prometheus as a different flavor of time-series database
+- Nginx is used as a proxy (mostly to fix CORS issue)
+
+Testing locally
+===============
+
+You can set-up your own local grafana-stack easily. In order to do that, you
+probably don't need to set-up passwords or even the `nginx` proxy.
```
Comment (spxtr) [inline velodrome/grafana-stack/README.md]: nit: It's not much help to say that I probably don't need to do something. Do I or don't I?

File: velodrome/fetcher/README.md (hunk 1 above)
Comment (apelisse) [inline velodrome/fetcher/README.md]: This one:

https://api.github.com/repos/kubernetes/kubernetes/issues/comments

I talked about that with an engineer from Github, looks like they fixed it now, great! We can definitely improve the code now :-)

File: velodrome/grafana-stack/README.md (hunk 2 above)
Comment (apelisse) [inline velodrome/grafana-stack/README.md]: You don't. What I meant is: "In production you NEED passwords and nginx. In local environment, you don't have to care.

File: velodrome/grafana-stack/README.md [hunk 3]
```diff
@@ -1,7 +1,29 @@
...
+
+You can run:
+
+```
+docker run -d -p 8083:8083 -p 8086:8086 tutum/influxdb:0.13
+docker run -i -p 3000:3000 grafana/grafana
+docker run -p 9090:9090 -v prom/prometheus
+```
+
+You will probably need to set-up the datasources/users into Grafana and InfluxDB as described
```
Comment (apelisse) [inline velodrome/grafana-stack/README.md]: You don't like "probably" ? I agree that it sounds like the docs doesn't even know what's going on :-)

File: velodrome/fetcher/README.md (hunk 1 above)
Comment (apelisse) [inline velodrome/fetcher/README.md]: I'm actually pretty excited it's fixed!

**Comment by apelisse** [comment]
PTAL @spxtr

**Comment by spxtr** [comment]
I think it's fine. Some grammar issues but nothing impeding readability. Usually it's a good idea to just check in the docs and then have other people try to use your system and update the docs as they have confusion :)

**Comment by apelisse** [comment]
Sure, but I may not be here to fix the confusion when needed ;-) So better be safe than sorry on this one.

Thanks!

**Comment by apelisse** [comment]
You can send as many PR as you want to fix grammar by editing readme from the github gui, I'd appreciate it :-)

**Comment by spxtr** [comment]
SGTM

**Pull request merged**